### ISSUE TITLE
dynatrace is not showing Application traces in Kubernetes

### ISSUE THREAD

**Issue by vivuu1989** [Dynatrace/dynatrace-operator#1047]
I was trying  toinstall and enable dynatrace installation in kubernetes pods and Azure Appservice applications.

Our Dynatrace server is hosted in onpremise and we have activegate instance in cloud and from AKS clusters subnet, port 9999 is opened to the activegate . And from ActiveGate to dynatrace server the port 443 is opened.

By following the  "Set up monitoring via Dynatrace Operator" using the manual mode 
 "classicFullStack Installation" we could add the kubernetes API to the dynatrace and able to monitor the infrastructure component like nodes cpu and memory resources and running workloads etc.

But actually we were looking to get the application traces of apps which are running kubernetes pods , like backend service callings, services response time, traffic flow etc, and etc..

But after installing the operator in AKS, we were not able to get these pods traces in dynatrace.
So is the dynatrace oneagent installation as daemonset is required for that?  I followed the [document][2]
 to install the dynatrace oneagent installation in AKS and pointed script installer url in the yaml file. But as per the document , the script installer is pointed to the dynatraceservers hostname port 443 and which is not opened from our clusters subnet directly, where as we have activegate instance in between dyatrace server and aks cluster. So is there any way to use activegate url instead of the dynatraceserver to install the oneagent in pods and get the all application related traces.

Searched everywhere and couldnt find a proper documents which explains the step by step guide for enabling the dynatrace monitoring for aks hosted application pods.

Also please suggest a way to monotor the azure appservice hosted application monotor also.

  [1]: https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-container-platforms/kubernetes/get-started-with-kubernetes-monitoring#manual
  [2]: https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-container-platforms/kubernetes/get-started-with-kubernetes-monitoring/set-up-k8s-monitoring-daemonset

**Comment by vivuu1989** [comment]
Since through activegate it didnt work as we expected, we opened the connectivity from AKS cluster subnet to the Dynatrace server over port 443  and installed the Dynatrace operator. But still we are not getting the application traces in the Dynatrace.


The error stating as below
lastTransitionTime: "2022-08-09T19:39:54Z"
    message: 'error when querying token on secret dynatrace:xxxxxxx: invalid
      character ''<'' looking for beginning of value'

I decripted the secret key and crossverified the API token, which is correct and not sure why this error is coming. Due to this the oneagent nodes are still not rolledout in the cluster.
